### PR TITLE
Add noorul json patient edit

### PR DESCRIPTION
**Pull request by itssodium**
Some of my functionality disappeared upon merging.

### CODE REVIEW

File: src/main/java/seedu/address/model/Model.java [hunk 1]
```diff
@@ -171,4 +171,9 @@
      * @param room The room from which the task should be delete.
      */
     void deleteTaskFromRoom(Task task, Room room);
+
```
Comment (raymondge) [inline src/main/java/seedu/address/model/Model.java:179]: Maybe consider "returns" instead of "gets" for javadoc consistency?

**Review by raymondge — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/model/Model.java (hunk 1 above)
Comment (itssodium) [reply src/main/java/seedu/address/model/Model.java:179]: Okay!

**Review by chiamyunqing — APPROVED** [review]
LGTM

**Pull request merged**